internal/guest/bridge/bridge_v2.go

```diff
@@ -317,35 +315,20 @@ func (b *Bridge) getPropertiesV2(r *Request) (_ RequestResponse, err error) {
 		return nil, errors.New("getPropertiesV2 is not supported against the UVM")
 	}
 
-	c, err := b.hostState.GetCreatedContainer(request.ContainerID)
+	properties, err := b.hostState.GetProperties(ctx, request.ContainerID, query)
 	if err != nil {
 		return nil, err
 	}
 
-	for _, requestedProperty := range query.PropertyTypes {
-		if requestedProperty == prot.PtProcessList {
-			pids, err := c.GetAllProcessPids(ctx)
-			if err != nil {
-				return nil, err
-			}
-			properties.ProcessList = make([]prot.ProcessDetails, len(pids))
-			for i, pid := range pids {
-				properties.ProcessList[i].ProcessID = uint32(pid)
-			}
-		} else if requestedProperty == prot.PtStatistics {
-			cgroupMetrics, err := c.GetStats(ctx)
-			if err != nil {
-				return nil, err
-			}
-			properties.Metrics = cgroupMetrics
+	propertyJSON := []byte("{}")
```

Comment:
you can actually marshal `nil` values as well 😄 something that I've recently learned.

Reply:
Thanks for the tip!

Reply:
I might be misremembering this though... let me double check lol